crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -1218,9 +1218,10 @@
   ],
   [
     "0",
-    "IF 0xbd ELSE 1 ENDIF",
+    "IF 0xc4 ELSE 1 ENDIF",
     "",
-    "OK"
+    "OK",
+    "opcodes above OpTxOutputSpk invalid if executed"
```

Comment:
Since this test passes, shouldn't it be clearer if it says "opcodes above OpTxOutputSpk are invalid **only** if executed" (or maybe I don't understand something?)